simphony_mayavi/plugins/engine_wrappers/loader.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import importlib
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def get_factories(submodules=("lammps_md", "kratos", "openfoam", "jyulb"),
```

Comment:
I would suggest moving the wrappers tuple as a `DEFAULT_ENGINES` contant and defaulting to `submodules=DEFAULT_ENGINES`